database/sqlite/sqlite_functions.c

```diff
@@ -62,6 +62,7 @@ const char *database_cleanup[] = {
     "delete from chart where chart_id not in (select chart_id from dimension);",
```

Comment:
I already approved the PR, because the goal was reached according the tests I ran. A small suggestion that can be address in another PR, this pointer has SQ keywords in lowercase and the last is in uppercase, considering that we have other pointers with SQL keyword  in uppercase, my suggestion is also convert the lowercase here to uppercase.